qiskit/compiler/transpiler.py

```diff
@@ -269,6 +270,11 @@ def callback_func(**kwargs):
             plugin is not used. You can see a list of installed plugins by
             using :func:`~.list_stage_plugins` with ``"optimization"`` for the
             ``stage_name`` argument.
+        ignore_backend_default_methods: If set to ``True`` any default methods specified by
+            a backend will be ignored. Some backends specify alternative default methods
+            to support custom compilation target passes/plugins which support backend
+            specific compilation techniques. If you'd prefer that these did not run and
+            use the defaults for a given optimization level this can be used.
```

Comment:
I'm open to changing the name, what about `ignore_backend_supplied_default_methods`? I'm hesitant about the two suggestions because the plugin could be defined or supplied from somewhere else. For example, I could see a backend using the `synthesis` translation method (assuming it actually worked reliably which I'm skeptical of) which is supplied and defined in terra itself. Or more realistically a plugin defined in a different package

Reply:
Ok. I see that it's also clear in the doc that it means "instance method". It makes sense to me that whatever the name of the method in question here (eg `gnore_backend_supplied_default_methods`) , including the word "method" makes sense because it controls which methods will be called.
(I don't know why my comment appears twice in my view of this page. Probably due to the way I finished the review.)

Reply:
I renamed it to `ignore_backend_supplied_default_methods` in https://github.com/Qiskit/qiskit-terra/pull/8648/commits/d0040806e6ee8315bed57400dc7a216f19c2d41f